scripts/generate_v5_configuration.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs');
+require('dotenv').config();
+const defaultConfigJson = require('../config/config.json');
+
+const nodercConfigPath = '/ot-node/.origintrail_noderc';
+const v5ConfigPath = '/ot-node/data/.v5_configuration';
+
+try {
+    console.log('Starting configuration update');
+    const defaultConfig = defaultConfigJson[process.env.NODE_ENV];
+    const configFile = fs.readFileSync(nodercConfigPath);
+    const config = JSON.parse(configFile);
+    const { blockchain } = config;
+    blockchain.blockchain_title = defaultConfig.blockchain.blockchain_title;
+    blockchain.network_id = defaultConfig.blockchain.network_id;
+    blockchain.node_wallet = config.node_wallet;
+    delete config.node_wallet;
+    blockchain.node_private_key = config.node_private_key;
+    delete config.node_private_key;
+    blockchain.management_wallet = config.management_wallet;
+    delete config.management_wallet;
+    if (config.erc725_identity_filepath) {
+        blockchain.identity_filepath = config.erc725_identity_filepath;
+        delete config.erc725_identity_filepath;
+    } else {
+        blockchain.identity_filepath = defaultConfig.erc725_identity_filepath;
+    }
+    delete config.blockchain;
```

Comment:
We should move/delete gas_limit, gas_price, hub_contract_address, rpc_server_url

Reply:
Please check line: 13. If user provided some of those values they will be used. If not default one will be used after v5 upgrade.